packages/ocaml-core/build.sh

```diff
@@ -0,0 +1,11 @@
+TERMUX_PKG_HOMEPAGE=https://ocaml.org/
+TERMUX_PKG_DESCRIPTION="OCaml is an industrial-strength programming language supporting functional, imperative and object-oriented styles"
+TERMUX_PKG_LICENSE="LGPL"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=4.12.0
+TERMUX_PKG_LICENSE_FILE="LICENSE"
```

Comment:
Should be no need to specify the license file in this case

Reply:
https://github.com/termux/termux-packages/blob/4f3f1ee43f30c4975dd9893a2067af0fd0db8156/scripts/build/termux_step_install_license.sh#L28-35

It is needed for LGPL

Reply:
We have generic license files for LGPL in termux-licenses: https://github.com/termux/termux-packages/tree/master/packages/termux-licenses/LICENSES. But only for LGPL-{2.0,2.1,3.0}, "LGPL" is not recognised. LICENSE should be LGPL-2.1, based on https://github.com/ocaml/ocaml/blob/trunk/LICENSE

Reply:
They do have a paragraph about "special exception" in their license file though, so including it as was done from the beginning is probably better. Please change to:
```
TERMUX_PKG_LICENSE=LGPL-2.1
TERMUX_PKG_LICENSE_FILE=LICENSE
```